contentcuration/contentcuration/models.py

```diff
@@ -1745,6 +1755,14 @@ class Meta:
         indexes = [
             models.Index(fields=["node_id"], name=NODE_ID_INDEX_NAME),
             models.Index(fields=["-modified"], name=NODE_MODIFIED_DESC_INDEX_NAME),
+            models.Index(fields=["grade_level_labels"], name=GRADE_LEVEL_LABEL_INDEX),
+            models.Index(
+                fields=["resource_type_labels"], name=RESOURCE_TYPE_LABEL_INDEX
+            ),
+            models.Index(
+                fields=["learning_activity_labels"], name=LEARNING_ACTIVITY_LABEL_INDEX
+            ),
+            models.Index(fields=["category_labels"], name=CATEGORY_LABEL_INDEX),
```

Comment:
Could you remind me, did you find any benefit with these indices? If we go with a bitmask approach for searching, then perhaps we can avoid adding these and prevent the performance hit on writing to the table.